zerver/lib/webhooks/common.py

```diff
@@ -114,3 +130,38 @@ def validate_extract_webhook_http_header(request: HttpRequest, header: str,
         raise MissingHTTPEventHeader(header)
 
     return extracted_header
+
+class AuthStrategies:
+    SIMPLE_TOKEN = 1
+
+def http_api_callback(bot: UserProfile, auth_strategy: int,
+                      callback_url: str) -> Optional[Dict[str, Any]]:
+    """
+        The way we contact the callback endpoint will vary based on the
+        authentication method required to contact it.
+
+        For SIMPLE_TOKEN based auth, we can simply modify the url of the
+        endpoint accordingly.
+    """
```

Comment:
Not a huge deal but the docstring's contents do not need to be indented here and I'm pretty sure we don't indent our docstrings throughout the code. :)

Reply:
Oh, I just realised that. I kinda indented that way out of habit, I'll make the change. :)